bitcoin/src/taproot.rs

```diff
@@ -619,6 +621,8 @@ pub enum HiddenNodes {
     /// Indicates an attempt to construct a tap tree from a builder containing hidden parts.
     HiddenParts(NodeInfo),
 }
+#[cfg(feature = "trait-checks")]
+internals::check_pub_error!(HiddenNodes);
```

Comment:
Unrelated but it looks like this should be called `HiddenNodesError` or be withing an `error` module.

Reply:
Noted, thanks.